### PR TITLE
Add Slexy.org translator

### PR DESCRIPTION
**Pull request by febrezo**
Adding a wrapper to process Slexy.org pastes with a snapshot. Included metadata: 
- Author
- Date
- Title

### CODE REVIEW

File: Slexy.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	// Adding the attachment
+	newItem.attachments.push({
+		title: "Slexy Snapshot",
+		mimeType: "text/html",
+		url: url
+	});	
+	
+	newItem.complete();
+}/** BEGIN TEST CASES **/
+var testCases = [
```
Comment (adam3smith) [inline Slexy.js]: please create new tests for Slexy and delete the pastebin tests here.

Comment (adam3smith) [inline Slexy.js]: please create new tests for Slexy and delete the pastebin tests here.

Comment (adam3smith) [inline Slexy.js]: please create new tests for Slexy and delete the pastebin tests here.

Comment (febrezo) [inline Slexy.js]: Added and fixed. It was a mistake when branching.

File: Slexy.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	// Creating the item
+	var newItem = new Zotero.Item(resourceType);
+
+	//Setting common data:
+	newItem.websiteTitle = "Slexy";
+	newItem.websiteType = "Paste Site";
+	newItem.url = url;
+	
+	// Grabbing paste's metadata from "data" object
+	var info = ZU.xpathText(doc, '//table[@id="data"]//b').split(',');
```
Comment (adam3smith) [inline Slexy.js]: That's unnecessarily fragile.
You can just go through the four table cells:
`//table[@id="data"]/tbody/tr[1]/td[1]/b`    (and 1/2, 2/1, 2/2 for tr and td respectively) to get at the four entries. That way, e.g. they don't break when one contains a comma.

File: Slexy.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	newItem.url = url;
+	
+	// Grabbing paste's metadata from "data" object
+	var info = ZU.xpathText(doc, '//table[@id="data"]//b').split(',');	
+	// Author
+	newItem.creators = [Zotero.Utilities.cleanAuthor(info[0], "author", false)];
+	newItem.title = info[2];
+	// Date: we will need some extra processign for the date
+	var aux = info[3];
+	var strDate = aux.substring(1,aux.lastIndexOf(" "));
```
Comment (adam3smith) [inline Slexy.js]: the timesamps are already in ISO format. Just read them in as they are.

File: Slexy.js (hunk 1 above)
Comment (adam3smith) [inline Slexy.js]: This no longer exists -- is there a way you can find something that's likely to remain stable (preferrably something that isn't spam?)

**Comment by AbeJellinek** [comment]
[Slexy has shut down and will delete existing pastes soon](https://slexy.org/), and this translator hasn't seen activity since 2016 (or fixes for some of @adam3smith's review comments). Closing.

**Pull request closed without merging**